static/app/components/searchSyntax/renderer.tsx

```diff
@@ -218,21 +218,32 @@ const InList = styled('span')`
   }
 `;
 
-const LogicGroup = styled('span')`
-  &:before,
-  &:after {
+const LogicGroup = styled(({children, ...props}) => (
+  <span {...props}>
+    <span>(</span>
```

Comment:
do you need a `span` element here?

Reply:
Yeah since I am using a pseudo element on top of this, where the span is just used to take up space.